### PR TITLE
Switching to use myusa and adding a manifest.

### PR DESCRIPTION
**Pull request by DavidEBest**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -25,16 +26,31 @@ def initialize
...
+    if authed?
+      erb :index, locals: { projects: key_list }
+    else
+      redirect '/auth/myusa'
+    end
+  end
+
+  get '/auth/myusa/callback' do
+    if env['omniauth.auth']
+      session[:authenticated] = true
```
Comment (afeld) [inline app.rb]: The session is secure, right? i.e. someone couldn't just set this themselves? Also, I'd maybe store something like `user_email` instead, which I could see being useful in the future.

**Comment by afeld** [comment]
Mind adding instructions about how to run the app, including things like the required environment variables for MyUSA, and what org/space it's deployed to?

File: Procfile [hunk 2]
```diff
@@ -0,0 +1 @@
+web: rackup -p $PORT -E production
```
Comment (afeld) [inline Procfile]: I'd get rid of this file and move this to the `command` in the `manifest.yml`.

File: app.rb (hunk 1 above)
Comment (DavidEBest) [inline app.rb]: It is supposed to be secure, but I'll switch it out for `encrypted_cookie`, which is supposed to be more secure. Good call on the user_email bit, I'll add that.

**Comment by DavidEBest** [comment]
@afeld I addressed your comments. There isn't a publicly accessible version of the app yet, I need to go through the process of getting the MyUSA app 'public'.

File: app.rb [hunk 3]
```diff
@@ -7,9 +8,9 @@ class ComplianceData < Sinatra::Base
 
   config_file 'credentials.yml'
 
-  configure :production do
-    use Rack::Auth::Basic do |username, password|
-      username == settings.site_user && password == settings.site_password
+  helpers do
+    def authed?
+      session[:authenticated]
```
Comment (afeld) [inline app.rb]: Maybe just use `!!session[:user_email]`? Then there's no need for `session[:authenticated]`.

File: README.md [hunk 4]
```diff
@@ -2,6 +2,41 @@
...
+`APP_ID`: MyUSA Consumer Public Key 
+`APP_SECRET`: MyUSA Consumer Secret Key 
+`COOKIE_SECRET`: A secret for your encrypted cookies.
+
+You can generate a good secret with:
+```
+ruby -rsecurerandom -e "puts SecureRandom.hex(32)"
+```
+
+Run the application with `rackup`
```
Comment (afeld) [inline README.md]: Missing a period here.

**Comment by afeld** [comment]
Just two small things!

**Comment by DavidEBest** [comment]
@afeld Thanks for the feedback. I've addressed both of those things.

**Comment by afeld** [comment]
Needs manual merge, but otherwise, :shipit: !

**Pull request merged**